src/components/Profile/Profile.js

```diff
@@ -50,9 +51,42 @@ class UserProfile extends Component {
       }));
     }
   }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    console.log("submitt");
+    if ( this.state.newpassword!=="" && this.state.confnewpassword==="" ) {
+      alert("Please confirm your password !!!");
+      return;
+    }
+    if ( this.state.newpassword==="" && this.state.confnewpassword!=="" ) {
+      alert("Please enter both password fields !!!");
+      return;
+    }
+    if ( this.state.newpassword!==this.state.confnewpassword ) {
+      alert("Password do not match !!!");
+      return;
+    }
+    const update={};
+    const body={};
+    if ( this.state.name!=="" ) {
```

Comment:
Use `let body={}`
Check whether name and email are same or not before adding it to body.